src/actions/guides/database/database_setup.cr

```diff
@@ -41,18 +82,38 @@ class Guides::Database::DatabaseSetup < GuideAction
     * retry_attempts
     * retry_delay
 
-    To set the connection pool options, just append a query string to the end of the
-    `settings.url` option (e.g. `?initial_pool_size=5&retry_attempts=2`)
+    To set the connection pool options, just set the `query` option in your `Avram::Credentials` to a query string.
+    (e.g. `initial_pool_size=5&max_pool_size=10`).
 
-    ### Avram Configuration
+    > If using an environment variable, the query string should come at the end of your connection string.
+    > (e.g. `postg)
+
+    ### Apps not using Avram
+
+    Avram requires a `credentials` option to be set. If you decide to not use Avram as your ORM,
+    you can set this option to `Avram::Credentials.void`.
 
-    Avram requires a `url` option to be set. If you decide to not use Avram as your ORM,
-    you can set this option to a string like `"unused"`.
+    ```crystal
+    # An example can be found on this website's source
+    # https://github.com/luckyframework/website/blob/master/config/database.cr
+    AppDatabase.configure do |settings|
+      # No database is required
+      settings.credentials = Avram::Credentials.void
+    end
+
+    Avram.configure do |settings|
+      settings.database_to_migrate = AppDatabase
+    end
+    ```
 
     Optionally, the `lazy_load_enabled` is set to `false` for development and test.
     This causes Lucky to raise an exception if you forget to preload an association,
     but will not raise an exception in production.
 
+    ```crystal
+    settings.lazy_load_enabled = Lucky::Env.production?
+    ```
```

Comment:
Sounds like this should be under a new header since it is not part of  "Apps not using Avram"